src/components/Molecules/SearchResult/SearchResult.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import Text from '../../Atoms/Text/Text';
+import Picture from '../../Atoms/Picture/Picture';
+
+/**
+ * Search Result
+ */
+const Wrapper = styled.article`
+  width: 100%;
+  height: 100%;
+  background-color: #fff;
+`;
+
+const Link = styled.a`
+  display: flex;
+  height: 100%;
+  border: 0;
+  flex-direction: column;
+  text-decoration: none;
+  color: inherit;
+  width: 100%;
+  @media ${({ theme }) => theme.breakpoint('small')} {
+    flex-direction: row;
+  }
+`;
+
+const ImageWrapper = styled.div`
+  height: auto;
+  @media ${({ theme }) => theme.breakpoint('small')} {
+    width: ${({ smallImageWidth }) => smallImageWidth};
+  }
+  @media ${({ theme }) => theme.breakpoint('large')} {
+    width: ${({ largeImageWidth }) => largeImageWidth};
+  }
+`;
+
+const CopyWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0px 10px 0px 20px;
```

Comment:
Same here we can use our `${spacing()}`.

Reply:
done